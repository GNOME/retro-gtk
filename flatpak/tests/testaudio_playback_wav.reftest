[Retro Reftest]
Path=/testaudio_playback_wav
Core=/app/lib/libretro/testaudio_callback_libretro.so
Medias=BAK_1.wav;

[Options]

[Frame 0]
State=Refresh
Video=testaudio_callback.png
