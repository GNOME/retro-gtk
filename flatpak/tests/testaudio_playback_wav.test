[Retro Reftest]
Path=/testaudio_playback_wav
Core=/app/lib/libretro/testaudio_callback_libretro.so
Medias=BAK_1.wav;

[Frame 0]
Tests=Run;Video;
Video=testaudio_callback.png
