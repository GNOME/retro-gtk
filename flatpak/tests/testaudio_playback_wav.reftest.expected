/testaudio_playback_wav/Boot: OK
/testaudio_playback_wav/Options: OK
/testaudio_playback_wav/0/State Refresh: OK
/testaudio_playback_wav/0/Run: OK
/testaudio_playback_wav/0/Video: OK
